platforms/osx/src/osxPlatform.mm

```diff
@@ -93,18 +96,125 @@ void initGLExtensions() {
     return data;
 }
 
+std::vector<char> loadNSFont(NSFont* _font) {
+    if (!_font) {
+        return {};
+    }
+
+    CGFontRef fontRef = CGFontCreateWithFontName((CFStringRef)_font.fontName);
+
+    if (!fontRef) {
+        return {};
+    }
+
+    std::vector<char> data = [TGFontConverter fontDataForCGFont:fontRef];
+
+    CGFontRelease(fontRef);
+
+    if (data.empty()) {
+        LOG("CoreGraphics font failed to decode");
+    }
+
+    return data;
+}
+
+bool allowedFamily(NSString* familyName) {
```

Comment:
We could share the list of allowed families between iOS and OSX if those are the same.

Reply:
I will take care of this in the coretext usage work I am doing.